setup.py

```diff
@@ -47,6 +48,8 @@
         'test': [
           'plone.app.testing',
           'plone.app.dexterity',
+          'Products.ATContentTypes',
+          'unittest2',
```

Comment:
Plone 4.2 runs under Python 2.7; Python 2.6 should be deprecated on tests and people wanting to use it in production must be aware of that.